src/coreclr/vm/class.h

```diff
@@ -1694,7 +1698,12 @@ class EEClass // DO NOT CREATE A NEW EEClass USING NEW!
     PTR_MethodTable m_pMethodTable;
 
     PTR_FieldDesc m_pFieldDescList;
-    PTR_MethodDescChunk m_pChunks;
+
+    // Chain of MethodDescChunks. The "begin" pointer shouldn't be updated
+    // after being set. The "current" pointer will be updated as new MethodDescChunks
+    // are added.
+    PTR_MethodDescChunk m_pChunksBegin;
+    PTR_MethodDescChunk m_pChunksCurrent;
```

Comment:
Can we add new chunks at the beginning instead of adding this pointer here?

The order of these chunks should not matter.

Reply:
Ah ok, I forgot that we introduced a dependency from reflection in https://github.com/dotnet/runtime/pull/69506/files#diff-4cae1ac340986fa73c94616dbe7cdb205ea0c9bea8d2adc5671a2ca3f00d9ffaR2714-R2717